sdk/nodejs/output.ts

```diff
@@ -408,7 +458,7 @@ export function all<T>(val: Input<T>[] | Record<string, Input<T>>): Output<any>
         const allOutputs = val.map(v => output(v));
 
         const [resources, isKnown, isSecret] = getResourcesAndDetails(allOutputs);
-        const promisedArray = Promise.all(allOutputs.map(o => o.promise()));
+        const promisedArray = Promise.all(allOutputs.map(o => <any>(<any>o).promise(/*withUnknowns*/ true)));
```

Comment:
so: `promise` is an internal API for us.  If we could see it before, we should be able to now see that it takes an optoinal boolean.  So it doesn't feel like the inner `<any>` casts should be necesarry.